rkt/metadata_service.go

```diff
@@ -511,6 +533,7 @@ func handlePodSign(w http.ResponseWriter, r *http.Request) {
 func handlePodVerify(w http.ResponseWriter, r *http.Request) {
 	defer r.Body.Close()
 
+	w.Header().Add("Content-Type", "text/plain; charset=us-ascii")
```

Comment:
Is this really ascii? Actually as far as I can see here in this function, we don't write anything to the response body.